plugins/zfs/zfs_count

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#%# family=auto
+#%# capabilities=autoconf
+
+# Michael Grote
+# michael.grote ät posteo.de
+# Outputs the count of zfs pools, datasets and snapshots.
+
+# lese alle pools, sed löscht die erste zeile
+list_pools=$(zpool list | sed 1d)
+
+# wenn parameter = ...
+if [ "$1" = "autoconf" ]; then
```

Comment:
Here you should verify, that `zpool` is available.

Reply:
added